src/Tizen.NUI.Components/Style/SliderStyle.cs

```diff
@@ -227,6 +227,15 @@ public Extents TrackPadding
             set => SetValue(TrackPaddingProperty, value);
         }
 
+        /// <summary>
+        /// Gets or sets the maximum size of the thumb.
+        /// This is used to calculate the track length and thumb position.
+        /// The track length is calculated by subtracting the maximum size of the thumb from the slider size.
+        /// The thumb position is calculated by using the track length.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Size ThumbMaximumSize { get; set; } = new Size();
```

Comment:
Just my curiosity :
Without bindableproperty, is it possible to declare `new` as Size constructor?